### PR TITLE
Snses without actionables banner

### PR DESCRIPTION
**Pull request by mstrasinskis**
# Motivation

Because not all projects support actionable proposals (return neuron ballots), not all actionable proposals will be shown on the actionable proposals page. To make it clear that there could be more votable proposals, we display a banner with a list of projects that do not have actionable support. In this PR, we create a banner component that will be shown on the actionable proposals page.

# Changes

- Add new component `ActionableProposalsNotSupportedSnses`

# Tests

- PO for `ActionableProposalsNotSupportedSnses`
- Tests for `ActionableProposalsNotSupportedSnses`

# Todos

- [ ] Add entry to changelog (if necessary).
not yet.

# Screenshot

image

### CODE REVIEW

File: frontend/src/lib/components/proposals/ActionableProposalsNotSupportedSnses.svelte [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+        >{replacePlaceholders(
+          $i18n.actionable_proposals_not_supported_snses.title,
+          {
+            $snsNames: snsNames,
+          }
+        )}</svelte:fragment
+      >
+      <p class="description" slot="description">
+        {replacePlaceholders(
+          $i18n.actionable_proposals_not_supported_snses.text,
```
Comment (dskloetd) [inline frontend/src/lib/components/proposals/ActionableProposalsNotSupportedSnses.svelte:28]: There's nothing to replace here, right?

File: frontend/src/tests/lib/components/proposals/ActionableProposalsNotSupportedSnses.spec.ts [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+  it("should not render the banne when there are no Snses", async () => {
+    const po = renderComponent();
+    expect(await po.getBannerPo().isPresent()).toEqual(false);
+  });
+
+  it("should display title", async () => {
+    addSnsesWithSupport([false, false]);
+    const po = renderComponent();
+    expect(await po.getBannerPo().getTitleText()).toEqual(
+      "Catalyze, Catalyze don’t yet support actionable proposals."
```
Comment (dskloetd) [inline frontend/src/tests/lib/components/proposals/ActionableProposalsNotSupportedSnses.spec.ts:63]: Please provide SNS names inside this test file.
As a rule, we shouldn't expect on mock data that's defined somewhere else.

File: frontend/src/tests/lib/derived/actionable-proposals.derived.spec.ts [hunk 3]
```diff
@@ -282,4 +283,61 @@ describe("actionable proposals derived stores", () => {
...
+      actionableSnsProposalsStore.set({
+        rootCanisterId: principal0,
+        proposals: [],
+        includeBallotsByCaller: false,
+      });
+      expect(
+        get(actionableProposalNotSupportedUniversesStore).map(
+          ({ canisterId }) => canisterId
+        )
+      ).toEqual([principal0.toText()]);
```
Comment (dskloetd) [inline frontend/src/tests/lib/derived/actionable-proposals.derived.spec.ts:317]: Is the conversion to text necessary?
Does it not work if you remove the conversion to string on line 315 and then expect `.toEqual([principal0])`?

File: frontend/src/lib/components/proposals/ActionableProposalsNotSupportedSnses.svelte (hunk 1 above)
Comment (mstrasinskis) [reply frontend/src/lib/components/proposals/ActionableProposalsNotSupportedSnses.svelte:28]: Good catch!

File: frontend/src/tests/lib/components/proposals/ActionableProposalsNotSupportedSnses.spec.ts (hunk 2 above)
Comment (mstrasinskis) [reply frontend/src/tests/lib/components/proposals/ActionableProposalsNotSupportedSnses.spec.ts:63]: Yep, done.

File: frontend/src/tests/lib/components/proposals/ActionableProposalsNotSupportedSnses.spec.ts [hunk 4]
```diff
@@ -0,0 +1,75 @@
...
+    );
+  });
+
+  it("should display description", async () => {
+    addSnsesWithSupport([false]);
+    const po = renderComponent();
+    expect(await po.getBannerPo().getDescriptionText()).toEqual(
+      "If an SNS DAO wishes to support actionable proposals, need to upgrade to a newer version of the SNS governance canister."
+    );
+  });
```
Comment (dskloetd) [inline frontend/src/tests/lib/components/proposals/ActionableProposalsNotSupportedSnses.spec.ts:74]: Can you add a test with a mix of supporting and non-supporting sns'es?

**Review by dskloetd — APPROVED** [review]

File: frontend/src/tests/lib/components/proposals/ActionableProposalsNotSupportedSnses.spec.ts (hunk 4 above)
Comment (mstrasinskis) [reply frontend/src/tests/lib/components/proposals/ActionableProposalsNotSupportedSnses.spec.ts:74]: Good point, updated.

**Pull request merged**